app/ee/omnichannel/containers/OmnichannelHeader/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { memo, useEffect, useState } from 'react';
+import { Switch, View } from 'react-native';
+
+import * as List from '../../../../containers/List';
+import styles from './styles';
+import { SWITCH_TRACK_COLOR, themes } from '../../../../lib/constants';
+import { useTheme } from '../../../../theme';
+import RocketChat from '../../../../lib/rocketchat';
+import { IUser } from '../../../../definitions/IUser';
+import { showConfirmationAlert } from '../../../../utils/info';
+import I18n from '../../../../i18n';
+import { changeLivechatStatus, isOmnichannelStatusAvailable } from '../../lib';
+import OmnichannelQueue from './OmnichannelQueue';
+
+interface IOmnichannelStatus {
+	searching: boolean;
+	goQueue: () => void;
+	queueSize: number;
+	inquiryEnabled: boolean;
+	user: IUser;
+}
+
+const OmnichannelStatus = memo(({ searching, goQueue, queueSize, user }: IOmnichannelStatus) => {
+	const { theme } = useTheme();
+	const [status, setStatus] = useState(isOmnichannelStatusAvailable(user));
+
+	if (searching || !(RocketChat.isOmnichannelModuleAvailable() && user?.roles?.includes('livechat-agent'))) {
+		return null;
+	}
+
+	useEffect(() => {
```

Comment:
👀

Reply:
React Hook "useEffect" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?